src/Workspaces/Core/Portable/Simplification/AbstractSimplificationService.cs

```diff
@@ -166,6 +165,9 @@ bool isNodeOrTokenOutsideSimplifySpans(SyntaxNodeOrToken nodeOrToken)
         }
 
         return document;
+
+        bool IsNodeOrTokenOutsideSimplifySpans(SyntaxNodeOrToken nodeOrToken)
```

Comment:
>IsNodeOrTokenOutsideSimplifySpans [](http://example.com/codeflow?start=13&length=33)

nit: just noticed the executeSerially flag is just supposed to be a debugger friendly bit, but it complicates things below. I'd be totally ok with deleting that as I think it actually complicates the code more than it helps.

Reply:
can do later (it's already being touched in https://github.com/dotnet/roslyn/pull/73471)